dist/dist-packages/linux/openziti-controller/env

```diff
@@ -0,0 +1,62 @@
+#
+# this is a systemd env file allowing simple assignments for ziti-controller.service environment
+#
+
+ZITI_CONTROLLER_RUN_ARGS="--log-formatter text"
+# ZITI_CONTROLLER_RUN_ARGS="--log-formatter text --verbose"
+
+# disable JSON logging during bootstrapping
+PFXLOG_NO_JSON=true
+
+#
+# for "ziti pki" and "ziti create config controller" commands in bootstrap.bash
+#
+
+# the advertised address of the controller is a domain name that can be resolved by all devices (default: hostname -f)
+ZITI_CONTROLLER_ADVERTISED_ADDRESS=
```

Comment:
use ZITI_CTRL_* -- they're meant for the same sort of precedence purpose and work that way